Makefile

```diff
@@ -16,6 +16,7 @@ $(NAME): cmake_build
 	@echo ${STY_GRE}${STY_BOL} "✅ Build Complete" ${STY_RES}
 	@echo ${STY_BLU} "➜ Project: " ${STY_WHI}${PROJECT}${STY_RES}
 	@echo ${STY_BLU} "➜ Status: " ${STY_GRE} "Mandatory" ${STY_RES}
+	@mv $(NAME) .
```

Comment:
@LeaYeh I swear you did that too, but on which branch?

Reply:
In your dream branch